webapp-django/crashstats/monitoring/urls.py

```diff
@@ -3,14 +3,20 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from django.conf.urls import url
+from django.views.generic import RedirectView
 
 from crashstats.monitoring import views
 
 
 app_name = 'monitoring'
 urlpatterns = [
     url(r'^monitoring/$', views.index, name='index'),
-    url(r'^monitoring/crontabber/$', views.crontabber_status, name='crontabber_status'),
+    url(r'^monitoring/cron/$', views.cron_status, name='cron_status'),
+    # FIXME(willkg): DEPRECATED AS OF 4/30/2019
```

Comment:
I guess I don't need to shout here.

Reply:
Relatedly, we have a bunch of "redirect deprecated thing to new thing" things in the webapp. We should do a scrub pass and remove existing cruft.